### PR TITLE
Raises the world fps to 40

### PR DESCRIPTION
**Pull request by Couls**
This is a change I want to make for pixel movement PR #52478 This feels almost necessary for smooth movement without stutter or jitter
The reason i'm making this PR is to get all the information and figure out whether or not this would be possible and whether or not its a good or bad idea
40 is the recommended world fps for pixel movement projects and what it defaults to when you create a new environment currently, 30 might work as a middle ground but i'm unsure.
We MIGHT be able to continue to work with 20 fps but it won't be optimal

@MrStonedOne I am requesting your input on this

Would this be possible? What are the main risks and how can we mitigate them?

### CODE REVIEW

**Comment by MrStonedOne** [comment]
It would seem a bunch of info is missing from the mc tab on the new browser based statpanels, more specifically, in the mc line of the mc tab, and the byond internal tick usage and the mc process limit items.

This is important to this pr, because that first number told us how long byond is taking every tick to send map/world updates to players.

Before the new panels how ever, I remember that number getting up to 80% on basil during high pop. meaning at 50ms ticks, byond would take 40ms to send map/world updates to clients, leaving only 10ms for any game code to run each tick.

Some of that wouldn't translate to the same amount of time at 40fps, as there would be less to do each tick, but some of it *would*, and that will likely prohibit increasing the fps.

**Comment by MrStonedOne** [comment]
also, a note, the fps is config item if you want to raise the default, you have to raise it in the config too.

File: config/config.txt [hunk 1]
```diff
@@ -286,10 +286,10 @@ TICK_LIMIT_MC_INIT 500
 
 ##Defines the ticklag for the world. Ticklag is the amount of time between game ticks (aka byond ticks) (in 1/10ths of a second).
 ##	This also controls the client network update rate, as well as the default client fps
-TICKLAG 0.5
+#TICKLAG 0.5
```
Comment (Cyprex) [inline config/config.txt:289]: The value needs to be updated too so that the comment below is still valid

**Review by SpaceManiac — CHANGES_REQUESTED** [review]
It sounds like 40fps capability on our live servers is not a given. Can the SSrunechat changes be split out so they can be merged separately, and 40fps tested on live before being made the default?

**Comment by Couls** [comment]
I'm with spacemaniac on this, I don't think pushing the fps to 40 is worth it or viable at the moment. Would've been nice though

**Pull request closed without merging**